tests/test_cache.py

```diff
@@ -26,6 +41,7 @@
 
 class TestCache(BaseTestCase):
     def test_load_from_cache_miss(self):
+        # Why is this ModelHandle needed?
```

Comment:
Was curious about this! Will remove this comment.

Reply:
Looks like I left this by mistake. Probably when I moved it over to a "constant" at the top 🤦 . You can remove.